retail/retail-java-applications/data-engineering-dept/business-logic/src/main/java/com/google/dataflow/sample/retail/businesslogic/core/utils/JSONUtils.java

```diff
@@ -104,9 +104,9 @@ public ConvertJSONtoPOJO(@Nullable String name, Class<T> clazz, SinkType sinkTyp
       PCollection<ErrorMsg> errorMsgs =
           result
               .getFailedToParseLines()
-              .apply("CreateErrorMessages", ParDo.of(new CreateErrorEvents(errMessageSchema)))
+              .apply("CreateRows", ParDo.of(new CreateErrorEvents(errMessageSchema)))
```

Comment:
Thanx for the catch, I think here this can give a little more info. So rather than CreateRows have something like CreateErrorRows.

Reply:
Done (new commit pushed to PR)